<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Course:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="course">
         <property name="minimumSize">
          <size>
           <width>235</width>
           <height>0</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Pic 10B. Intermediate Programming</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Pic 10C. Advanced Programming</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Hw 1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="hw1slide">
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="hw1spin">
             <property name="maximum">
              <number>100</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Hw 2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="hw2slide">
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="hw2spin">
             <property name="maximum">
              <number>100</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Hw 3</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="hw3slide">
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="hw3spin">
             <property name="maximum">
              <number>100</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Hw 4</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="hw4slide">
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="hw4spin">
             <property name="maximum">
              <number>100</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Hw 5</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="hw5slide">
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="hw5spin">
             <property name="maximum">
              <number>100</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Hw 6</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="hw6slide">
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="hw6spin">
             <property name="maximum">
              <number>100</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Hw 7</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="hw7slide">
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="hw7spin">
             <property name="maximum">
              <number>100</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Hw 8</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="hw8slide">
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="hw8spin">
             <property name="maximum">
              <number>100</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Midterm 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="mid1slide">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="mid1spin">
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Midterm 2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="mid2slide">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="mid2spin">
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Final Exam</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="finalslide">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="finalspin">
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton">
           <property name="text">
            <string>Schema A</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_2">
           <property name="text">
            <string>Schema B</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Overall Score:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>505</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGrade_Calculator">
    <property name="title">
     <string>Grade Calculator</string>
    </property>
   </widget>
   <addaction name="menuGrade_Calculator"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>hw1spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw1slide</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw2spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw2slide</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw3spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw3slide</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw4spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw4slide</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw5spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw5slide</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw6spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw6slide</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>393</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw7spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw7slide</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>452</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>453</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw8spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw8slide</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>511</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>512</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw1slide</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>hw1spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw2slide</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>hw2spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw3slide</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>hw3spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw4slide</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>hw4spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw5slide</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>hw5spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw6slide</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>hw6spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>393</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw7slide</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>hw7spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>452</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw8slide</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>hw8spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>512</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>511</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mid1spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mid1slide</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>487</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mid1slide</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>mid1spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>487</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mid2spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mid2slide</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>487</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mid2slide</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>mid2spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>487</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>finalspin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>finalslide</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>487</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>390</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>finalslide</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>finalspin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>383</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>408</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>finalspin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>finalslide</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>382</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
